jsx/Form.js

```diff
@@ -348,26 +351,41 @@ var TextboxElement = React.createClass({
       id: null,
       disabled: false,
       required: false,
+      hasError: false,
+      errorMessage: 'The field is required!',
```

Comment:
This default value should either be empty or "This input is invalid"

Reply:
it should just be empty if the hasError prop is removed

Reply:
I'm not even sure that there should be a default, but if there is I agree it should be empty.

Reply:
Agreed. I think I will put the default errorMessage to empty, simply because I find that having defaults really helps the developer to assess what props can be passed to a given component when using it.